EIPS/eip_195_pure_call.md

```diff
@@ -0,0 +1,23 @@
+### Parameters
+
+* BASE_GAS_COST: 80
+* CODE_BYTES_PER_GAS: 15
```

Comment:
Agreed.  Not sure what the right amount is, as the preprocessing can run from simple validation to full compilation.